docker-compose.yml

```diff
@@ -1,92 +1,73 @@
 services:
 
-  reverse-proxy:
```

Comment:
I would like to keep this part of the compose. It's here so that one can scale more web-server workers at once with `docker compose up --scale web-server=4` or whatever. If someone wanted to deploy 8Knot on bare metal this would be more convenient than having to reconfigure the gunicorn command.

Reply:
The reverse proxy is back